src/codegen/external_functions.rs

```diff
@@ -98,6 +111,20 @@ fn check_expression(expr: &Expression, call_list: &mut CallList) -> bool {
         | Expression::InternalFunction { function_no, .. } => {
             call_list.solidity.insert(*function_no);
         }
+        Expression::Builtin {
```

Comment:
Good catch, I thought there might be more

Reply:
That's why we have the assert